Analytics Plugin Documentation (#911)
elastic/search-ui: docs/api-plugins-analytics-plugin.mdx

---
id: api-plugins-analytics-plugin
slug: /search-ui/api/core/plugins/analytics-plugin
title: Analytics Plugin
date: 2023-01-05
tags: [""]
---

> The Analytics Plugin is a plugin that provides a way to send analytics events to the Behavioral Analytics Product. To use it, you must follow the instructions on how to set up the Analytics Product.

Reviewer: ```suggestion
Use the Analytics Plugin to send analytics events to the Behavioral Analytics Product. Follow the instructions to set up the plugin.
```

Minor rewording for simplicity

## Installation

```bash
yarn add @elastic/search-ui-analytics-plugin
## OR
npm install @elastic/search-ui-analytics-plugin
```

## Basic Usage

> Within your Search UI configuration, you can add the Analytics Plugin like so:

Reviewer: Is there a missing example here after "like so:"?

Author: the example is below doesn't follow immediately after. Ive updated it to be immediately after :)

> By default, the Analytics Plugin will use the Behavioral Analytics client thats provided when the integrated via script tag.

Reviewer: ```suggestion
By default, the Analytics Plugin will use the Behavioral Analytics client provided when integrated using a `script` tag.
```
